src/wallet/rpcwallet.cpp

```diff
@@ -2878,7 +2895,8 @@ static RPCHelpMan listunspent()
             + HelpExampleCli("listunspent", "6 9999999 '[]' true '{ \"minimumAmount\": 0.005 }'")
             + HelpExampleRpc("listunspent", "6, 9999999, [] , true, { \"minimumAmount\": 0.005 } ")
                 },
-        [&](const RPCHelpMan& self, const JSONRPCRequest& request) -> UniValue
+        [&](const RPCHelpMan& self, const JSONRPCRequest& request)
+            EXCLUSIVE_LOCKS_REQUIRED(!cs_main) -> UniValue
```

Comment:
The wallet shouldn't know and care about validation locks. Also, this only silences a warning, it won't and can't ever be checked upstream in the call graph

Reply:
Would `NO_THREAD_SAFETY_ANALYSIS` be better?

The purpose of this PR is to only silence the warnings.